### PR TITLE
Fix replay file processing

### PR DESCRIPTION
**Pull request by llucax**
- Really skip copying replay file if it doesn't exit
- Remove the `_output_dir` from the replay file
- Don't save the `_template` in the replay file if it looks local

### CODE REVIEW

**Comment by llucax** [comment]
Also minor, tested locally and I would like to move forward with migrating some projects, so will merge, but after-merged reviews are welcome!

File: cookiecutter/hooks/post_gen_project.py
Comment (daniel-zullo-frequenz) [inline cookiecutter/hooks/post_gen_project.py:165]: cosmetic: double `add`

File: cookiecutter/hooks/post_gen_project.py
Comment (daniel-zullo-frequenz) [inline cookiecutter/hooks/post_gen_project.py:151]:
```suggestion
        # that depends on the current environment and we don't want to add it as part of
```

File: cookiecutter/hooks/post_gen_project.py
Comment (daniel-zullo-frequenz) [inline cookiecutter/hooks/post_gen_project.py:163]:
```suggestion
                    "to be a remote repository, it won't be saved to avoid storing a local "
```

File: cookiecutter/hooks/post_gen_project.py
Comment (daniel-zullo-frequenz) [inline cookiecutter/hooks/post_gen_project.py:159]: Maybe you can do it in a line as follows (haven't tested it):
```python
prefixes = ("gh:", "git@", "https://")
if not template.startswith(prefixes):
```

**Review by daniel-zullo-frequenz — COMMENTED** [review]
I have a few cosmetic comments to check for

**Comment by llucax** [comment]
Thanks! All comments fixed!

**Comment by llucax** [comment]
Enabled auto-merge.

**Review by daniel-zullo-frequenz — APPROVED** [review]

**Pull request merged**
